src/main/groovy/com/zeroc/gradle/icebuilder/slice/SliceExtension.groovy

```diff
@@ -120,9 +120,12 @@ class SliceExtension {
                 def jarDirectories = [
                     [_iceHome, "share", "java"],                          // Default usr install
                     [_iceHome, _compat ? "java-compat" : "java", "lib"],  // Source distribution
-                    [_iceHome, "lib"]                                     // Opt style install & Windows distribution
                 ]
 
+                if(_iceHome != "/usr" && _iceHome != "/usr/local") {
```

Comment:
I don't think this is completely correct anymore as macOS with Apple silicon chips using Homebrew would have the `iceHome` in `/opt/homebrew`. 

The distinction between the two install styles is a bit of a pain.